build.gradle

```
@@ -68,6 +68,8 @@ ext {
     plumeScriptsHome = "${project(':checker').projectDir}/bin-devel/.plume-scripts"
     htmlToolsHome = "${project(':checker').projectDir}/bin-devel/.html-tools"
     doLikeJavacHome = "${project(':checker').projectDir}/bin/.do-like-javac"
+    templateforCheckerDir = "${parentDir}/templatefora-checker"
```

```suggestion
    templateforaCheckerDir = "${parentDir}/templatefora-checker"
```

For consistency with the directory and task.